containerz/containerz.proto

```diff
@@ -0,0 +1,286 @@
+//
+// Copyright 2023 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+// This file defines the gNOI APIs used to perform container operations on a
+// network device.
+syntax = "proto3";
+
+package gnoi.containerz;
+
+import "common/common.proto";
+import "google/rpc/status.proto";
+import "types/types.proto";
+
+option go_package = "github.com/openconfig/gnoi/containerz";
+
```

Comment:
Should we add a version tag here (like [this](https://github.com/openconfig/gnoi/blob/main/file/file.proto#L26))? I think it might also be worth adding a comment that this is a first pass at such an implementation.